pkg/manifest/loader/account.go

```diff
@@ -69,14 +66,51 @@ func parseSingleAccount(c *Context, a persistence.Account) (manifest.Account, er
 
 	acc := manifest.Account{
 		Name:        a.Name,
-		AccountUUID: accountId,
+		AccountUUID: accountUUID,
 		ApiUrl:      urlDef,
 		OAuth:       oAuthDef,
 	}
 
 	return acc, nil
 }
 
+func parseAccountUUID(u persistence.AccountUUID) (uuid.UUID, error) {
+	uuidValue, err := loadAccountUUID(u)
+	if err != nil {
+		return uuid.UUID{}, err
+	}
+
+	accountUUID, err := uuid.Parse(uuidValue)
+	if err != nil {
+		return uuid.UUID{}, invalidUUIDError{uuidValue, err}
+	}
+
+	return accountUUID, nil
+}
+
+func loadAccountUUID(u persistence.AccountUUID) (string, error) {
+	if u.Value == "" {
+		return "", errAccUidMissing
+	}
+
+	if u.Type == "" || u.Type == persistence.TypeValue { // shorthand or explicit type: value
+		return u.Value, nil
+	}
+
+	if u.Type == persistence.TypeEnvironment {
+		val, found := os.LookupEnv(u.Value)
+		if !found {
+			return "", fmt.Errorf("environment variable %q could not be found", u.Value)
+		}
+		if val == "" {
+			return "", fmt.Errorf("environment variable %q is defined but has no value", u.Value)
```

Comment:
not sure if we might want to use the `invalidUUIDError` here as well - semantically it seems to make sense, but IMO how it currently prints might look weird with an empty string